.travis.yml

```diff
@@ -29,15 +29,7 @@ matrix:
       env: PYTHON_VERSION=3.5
       language: generic
     - python: '3.5'
-      env: CONDA_BUILD=1.21.14
-      os: linux
-      language: python
-    - python: '3.5'
-      env: CONDA_BUILD=2.0.12
-      os: linux
-      language: python
-    - python: '3.5'
-      env: CONDA_BUILD=2.1.0
+      env: CONDA_BUILD=2.1.2
```

Comment:
Note this is the 4.2.x branch.  At this point, reality is we either get to "enforce passing" with testing against conda-build 2.1.2, or "accept variable failures" with testing against other conda-build versions.  For me, even though we have to tell users to upgrade conda-build, it feels better to "enforce passing."